### PR TITLE
mysql 8.4.2-icu75.1 -> 9.0.1

### PR DESCRIPTION
**Pull request by Zopolis4**
## Description
Updates mysql to 9.0.1, moves the icu version requirement to the depends_on call.

## Additional information
We should try and avoid using `min_glibc` unless it is 100% necessary, as this cuts off tons of devices. We should really only be using `min_glibc` on precompiled proprietary binary packages where we can't do anything about the glibc requirements. I'll go through packages and fix them, but i'd appreciate it if we stopped adding these requirements when there is no reason for them.

##
Builds:
- [x] `x86_64`
- [x] `i686`
- [x] `armv7l`
##
Tested & Working properly:
- [x] `x86_64`
- [x] `i686`
- [x] `armv7l` 

##
### Run the following to get this pull request's changes locally for testing.
```bash
CREW_REPO=https://github.com/Zopolis4/chromebrew.git CREW_BRANCH=1really crew update \
&& yes | crew upgrade
```

### CODE REVIEW

File: packages/mysql.rb [hunk 1]
```diff
@@ -3,22 +3,21 @@
...
   binary_sha256({
-     x86_64: '5ce6d8be4a2660d05b7ec4892b09f17a7236881fcc34f9756c6102c5d2539791'
+     x86_64: '78f197a0cc31d6657bc682cbbb5be958326872eef0b35d93d7633f59646265d8'
   })
 
   depends_on 'boost' => :build
   depends_on 'gcc_lib' # R
   depends_on 'glibc_lib' # R
-  depends_on 'icu4c' => :build
+  depends_on 'icu4c', '== 75.1'
```
Comment (uberhacker) [inline packages/mysql.rb:20]: Does this work?

File: packages/mysql.rb [hunk 2]
```diff
@@ -93,4 +84,10 @@ def self.postremove
         -DWITH_UNIT_TESTS=OFF" # Since we don't currently run the tests, there's no point in building them.
   # 52/136 Test  #52: merge_small_tests ..................................***Failed   62.42 sec
   # run_tests
+  cmake_install_extras do
+    FileUtils.mkdir_p "#{CREW_DEST_PREFIX}/etc/bash.d"
+    File.write "#{CREW_DEST_PREFIX}/etc/bash.d/01-mysql", <<~EOF
+      [ -x #{CREW_PREFIX}/bin/mysql.server ] && #{CREW_PREFIX}/bin/mysql.server start
+    EOF
+  end
```
Comment (uberhacker) [inline packages/mysql.rb:92]: Was this built with the lowest glibc container?

Comment (Zopolis4) [reply packages/mysql.rb:92]: It was built on the M90 container, which has glibc 2.27

File: packages/mysql.rb (hunk 1 above)
Comment (Zopolis4) [reply packages/mysql.rb:20]: yes

Comment (uberhacker) [reply packages/mysql.rb:20]: Great

File: packages/mysql.rb (hunk 2 above)
Comment (uberhacker) [reply packages/mysql.rb:92]: Excellent.

**Review by uberhacker — APPROVED** [review]

**Pull request merged**